LibreNMS/Interfaces/Polling/Sensors/WirelessLteCellPolling.php

```diff
@@ -0,0 +1,38 @@
+<?php
+/**
+ * WirelessCellPolling.php
```

Comment:
Oh, here is what is going on. 
Delete this file and just use WirelessCellPolling